cli/internal/run/run.go

```diff
@@ -1010,7 +1010,7 @@ func (e *execContext) exec(pt *packageTask, deps dag.Set) error {
 		outputs := pt.HashableOutputs()
 		targetLogger.Debug("caching output", "outputs", outputs)
 		ignore := []string{}
-		filesToBeCached := globby.GlobFiles(pt.pkg.Dir, outputs, ignore)
+		filesToBeCached := globby.GlobFiles(filepath.Join(e.rs.Opts.cwd, pt.pkg.Dir), outputs, ignore)
```

Comment:
This previously implicitly depended on `os.Getwd()` as called by `fs.Walk()`. That is not something implemented by any `afero` file system and enables configuring yourself into really weird states.

This change makes it explicit.

Reply:
This seems like a good change. It at least makes it clearer what kind of path we're passing in.